root_backend_checks.tf

```diff
@@ -221,21 +235,51 @@ module "statuses" {
 }
 
 module "yara_av_v2" {
-  source        = "./tdr-terraform-modules/generic_lambda"
-  tags          = local.common_tags
-  function_name = local.yara_av_v2_function_name
-  handler       = "matcher.matcher_lambda_handler"
+  source               = "./tdr-terraform-modules/generic_lambda"
+  tags                 = local.common_tags
+  function_name        = local.yara_av_v2_function_name
+  handler              = "matcher.matcher_lambda_handler"
+  reserved_concurrency = -1
   policies = {
-    "TDRYaraAVV2LambdaPolicy${title(local.environment)}" = templatefile("./templates/iam_policy/lambda_s3_only_policy.json.tpl", {
-      function_name = local.yara_av_v2_function_name,
-      account_id    = data.aws_caller_identity.current.account_id,
-      bucket_name   = local.upload_files_cloudfront_dirty_bucket_name
+    "TDRYaraAVV2LambdaPolicy${title(local.environment)}" = templatefile("./templates/iam_policy/lambda_av_policy.json.tpl", {
+      function_name     = local.yara_av_v2_function_name,
+      account_id        = data.aws_caller_identity.current.account_id,
+      dirty_bucket      = local.upload_files_cloudfront_dirty_bucket_name
+      clean_bucket      = module.upload_bucket.s3_bucket_name
+      quarantine_bucket = module.upload_bucket_quarantine.s3_bucket_name
     })
   }
   role_name = "TDRYaraAVV2LambdaRole${title(local.environment)}"
+  runtime   = "python3.7"
```

Comment:
Is it worth putting the different lambda runtimes (java, python) as locals, for consistency. I see one of the lambdas is on python 3.9, this one is on 3.7?

Reply:
Yeah OK, I will at some point try to get everything on 3.9 but that's a future problem. I've put a ticket in Jira.